recipes/marshmallow-oneofschema/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "marshmallow-oneofschema" %}
+{% set version = "2.0.0b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ca0464ffce7f442db2682e32b14ff1f8596dbee7ca97fe94fccc8e4c3cdc1343
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - marshmallow >=3.0.0b12,<4.0.0
+
+test:
+  imports:
+    - marshmallow_oneofschema
+
+about:
+  home: https://github.com/marshmallow-code/marshmallow-oneofschema
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Marshmallow library extension that allows schema (de)multiplexing'
+
+extra:
+  recipe-maintainers:
+    - dhirschfeld
+    - rzillman
```

Comment:
@rzillman Are you happy to be listed as a maintainer of this recipe?

Reply:
Yes I am @chrisburr